app/lib.ts

```diff
@@ -31,29 +31,33 @@ export { Accordion } from '../src/components/containers/Accordion/Accordion.tsx'
 export { Form } from '../src/components/forms/context/Form/Form.tsx';
 export { SubmitButton } from '../src/components/forms/context/SubmitButton/SubmitButton.tsx';
 
+// Forms > Common
+export { Label } from '../src/components/forms/common/Label/Label.tsx';
```

Comment:
minor nitpick: would be nice to maintain alphabetical order here and in the next section

Reply:
I'm trying to match this with the storybook ordering instead (so as in `preview.tsx`)